.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
```

Comment:
Before opening the PR I used `trusty` explicitly. I use `xenial` on another repo and it felt like the container was a bit faster to initialize.